### PR TITLE
Updates to hero graphic guidelines

### PR DESCRIPTION
**Pull request by nataliafitzgerald**
Short description explaining the high-level reason for the pull request

## Additions
 - Update documentation of min and max character counts for hero graphics #460 
 - Add full bleed illustration option to hero graphic guidelines #462
 - Add photo option to hero graphic guidelines #464
 - Update in-page navigation to align with other DM pages 

## Removals
- Remove the button and link options from the hero module #454

## Review

- @user

[Preview this PR without the whitespace changes](?w=0)

## Screenshots
screencapture-nataliafitzgerald-github-io-design-manual-global-elements-heroes-html-1491492748971

## Notes

-

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by nataliafitzgerald** [comment]
+ @sarahsimpson09 and @jordanafyne. I couldn't find you on the "Reviewers" list.

**Comment by jenn-franklin** [comment]
Looks good! Thanks for your work on this.

**Review by ajbush — APPROVED** [review]
Looks good to me!

**Comment by sarahsimpson09** [comment]
:+1:

**Comment by jordanafyne** [comment]
:+1:

File: global-elements/heroes.md [hunk 1]
```diff
@@ -33,75 +32,183 @@ Heroes function as a primary focal point on a page, often used to introduce a co
...
+<div class="content-33 content-last">
+##### Where to use
+[Landing page]({{site.baseurl}}/global-elements/landing-page.html)
 
-Heroes should be limited to first- (i.e., Consumer Help) or second-level (i.e., Credit & Credit Cards) landing pages. The fewer instances of heroes there are, the more impact they will convey. 
+[Sublanding page]({{site.baseurl}}/global-elements/sublanding-page.html)
+</div>
 
-Heroes should be the most prominent thing on the page, and help to establish visual hierarchy of where the page lives within the information architecture.
+<img alt="Image of hero graphic" src="../static/img/hero/hero_use_case.png"/>
```
Comment (jimmynotjim) [inline global-elements/heroes.md]: We've preferred using absolute urls, which would update the source here to `{{site.github.urll}}/static/img/hero/hero_use_case.png` (We should update all the existing `{{site.baseurl}}` uses but that's outside the scope of this PR)

File: global-elements/heroes.md [hunk 2]
```diff
@@ -33,75 +32,183 @@ Heroes function as a primary focal point on a page, often used to introduce a co
...
+
+<div class="content-50 content-last"> 
+  <p>Large screens (601+)</p>
+  <img alt="Image of bleed illustration hero graphic with dimensions" src="../static/img/hero/hero_style_variations_bleed_large.png"/> 
 </div>
 
-<div class="content-33">
-![Image of hero with call to action at mobile size]({{ site.baseurl }}/static/img/hero/Hero_mobile.png)
+<div class="content-50 content-last">
+ <p>Small screens (600-)</p>
```
Comment (jimmynotjim) [inline global-elements/heroes.md]: Indentation here should be two spaces.

**Review by jenn-franklin — APPROVED** [review]

**Review by jimmynotjim — APPROVED** [review]

File: global-elements/heroes.md [hunk 3]
```diff
@@ -33,75 +32,181 @@ Heroes function as a primary focal point on a page, often used to introduce a co
...
-Heroes can contain calls to action. Rules for button versus links apply in a call to action.
-</div>
+<h4>When to use</h4>
+- When orienting a user to a new section or topic
+- When the page serves as a navigational tool with general information about a subject and links to lower-level pages that contain more specific information
 
-<div class="content-33 content-last"></div>
-<h2 id="desktop">Style at desktop size<span class="cf-code-link"><a href="https://github.com/cfpb/capital-framework/blob/master/src/cf-layout/src/cf-layout.less#L618-L620">View code <span class="cf-icon cf-icon-external-link"></span></a></span></h2>
+<h4>When other options are better</h4>
+- When introducing a specific piece of content, like a blog, press release, or other lengthy or detailed content, use the [Text introduction]({{site.baseurl}}/global-elements/text-introduction.html)
```
Comment (ielerol) [inline global-elements/heroes.md]: change the link to point to text-introductions.html

**Review by ielerol — CHANGES_REQUESTED** [review]
looks good! just a broken link to fix

**Comment by nataliafitzgerald** [comment]
@ielerol - I fixed the broken link. Look good to go?

**Review by ielerol — APPROVED** [review]

**Review by kurzn — APPROVED** [review]
Looks good!

**Review by Scotchester — APPROVED** [review]
:+1: Big improvement. Thanks!

**Pull request merged**